Jellyfin.Server/Program.cs

```diff
@@ -86,6 +88,12 @@ private static async Task StartApp(StartupOptions options)
         {
             var stopWatch = new Stopwatch();
             stopWatch.Start();
+
+            // Log all uncaught exception to std error
+            static void UnhandledExceptionToConsole(object sender, UnhandledExceptionEventArgs e) =>
```

Comment:
Isn't the lambda supposed to go on the next line?

Reply:
It's not a lambda, it's an local function.